src/igvfd/schemas/gene.json

```diff
@@ -126,6 +126,52 @@
             },
             "permission": "admin_only"
         },
+        "collections": {
+            "title": "Collections",
+            "description": "Genes that are part of a particular data collections.",
+            "comment": "An array of research collections that is analyzing this gene.",
```

Comment:
seems we usually use the comment field to say "Do not submit. Collections are for DACC use only." in this case